README.md

```diff
@@ -426,6 +427,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [Setup Cassandra Schema](https://github.com/fabasoad/setup-cassandra-action) - Running scripts from the provided folder on top of Cassandra cluster.
 
+### Localization
+
+- [Find and automatically fix typos and grammar issues in your code](https://github.com/sobolevn/misspell-fixer-action) 
```

Comment:
I'm a little hesitant to add this to its own category. My reasoning is that the action in question here not only localizes, but fixes the typos and issues as well, which I would consider linting. Any thoughts on that?

Reply:
Also, lint dead-url check is temporarily disabled, so you will have to pull and push again.

Reply:
Yes, I agree with you hesitation. My thought was: linters are more like code oriented. When localization is more NL oriented. And that's why I made this decision. 

I guess it really belongs there.